EnglishWebRoot/school/script/load.js

```diff
@@ -1285,6 +1285,32 @@ function xmlLoaded(curXML) {
         // Should give enough time for the viewport meta to finally parse in the browser.
         $("body").css("display", "block");
         loadAssets();
+        // Fitting images to all the device according to their screen width and screen height 
+

```

Comment:
Remove unnecessary spaces if not required

Reply:
Removed spaces